### ISSUE TITLE
Performance of Parameter Efficient Fine Tuning Method

### ISSUE THREAD

**Issue by akash418** [malteos/finetune-evaluation-harness#8]
This post summarizes the performance of the big pre-trained models using a parameter efficient approach as mentioned here https://github.com/huggingface/peft. The results here have been documented for 4 types of peft techniques. For all the approaches after applying the parameter efficient reduction on the original model, the percentage of trainable parameters left was around 5-6%.

Task Description
Gnad10: https://huggingface.co/datasets/gnad10 (Sequence Classification)
Epochs: 1
Learning Rate: 3e-4

LORA params:
bias=none
r=8
lora alpha=0.16
lora dropout=0.0

P Tune params:
#virtual tokens=20
encoder hidden states=128

Prefix Tune params
#virtual tokens=20

Prompt Tune params:
#virtual tokens=10

| Model          | LORA | P tuning | Prefix tuning | Prompt tuning |
|----------------|------|----------|---------------|---------------|
| gpt2-xl-german | 0.13 | 0.13     | 0.24          | 0.13          |
| bloom-1b5-clp  | 0.28 | 0.12     | 0.28          | 0.12          |
| bloom-6b4-clp  | 0.32 | 0.16     | 0.53          | 0.14          |
